src/Input/Input.js

```diff
@@ -99,6 +155,15 @@ module.exports = kind(
 	/**
 	* @private
 	*/
+	create: function () {
+		Input.prototype.create.apply(this, arguments);
+		this.setAttribute('min', this.min);
```

Comment:
What do you think about changing these calls to
```
this.minChanged();
this.maxChanged();
```
to keep things DRY, and in case we decide to add additional logic to these change handlers?